critical_migrations.txt

```diff
@@ -0,0 +1 @@
+["20160225000000"]
```

Comment:
This shouldn't be here - maybe it was committed on accident from an upgrade on the git repo?